packages/quick_actions/quick_actions_ios/ios/Classes/ShortcutStateManaging.swift

```diff
@@ -0,0 +1,46 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import UIKit
+
+/// Manages the shortcut related states.
+protocol ShortcutStateManaging {
+
+  /// Sets the list of shortcut items.
+  ///
+  /// - Parameter items the list of shortcut items to be parsed and set.
+  func setShortcutItems(_ items: [[String: Any]])
+}
+
+/// A default implementation of `ShortcutStateManaging` protocol.
+final class DefaultShortcutStateManager: ShortcutStateManaging {
```

Comment:
Same here: i put this class in the same file as the protocol, but also ok to use separate files. 

More details discussed in [Rule 3(b) of the Swift POP naming convention proposal](https://docs.google.com/document/d/1H8F-0OmJ5GF4BCwfC_j7rs7k02dLkrcxpe2IZbaQg5Q/edit#heading=h.558lztygqmrp). Feel free to leave any comments.